yellowpages/yellowpages-war/src/main/webapp/yellowpages/jsp/contactManager.jsp

```diff
@@ -90,7 +90,10 @@ if (creation) {
 Form formUpdate = contactComplete.getUpdateForm();
 PagesContext context = (PagesContext) request.getAttribute("PagesContext");
 if (context != null) {
+  context.setCreation(false);
   context.setBorderPrinted(false);
+  if (creation)
+    context.setCreation(true);
```

Comment:
Code rules: the `if` clause should starts with `{` and ends with `}` even whether it contains only a single statement